.github/workflows/dev-green.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy to Dev-green
+
+on: workflow_dispatch
+
+jobs:
+  deploy:
+    name: Deploy
+    runs-on: ubuntu-latest
+    environment: dev-green
+    concurrency: dev-green
+    env:
+      AWS_REGION: us-east-1
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get commit metadata
+        run: |
+          echo SHA_SHORT=$(git rev-parse --short ${{ github.sha }}) >> $GITHUB_ENV
+          echo COMMIT_MSG=$(git log --format=%B -n 1 ${{ github.sha }} | head -n 1) >> $GITHUB_ENV
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.AWS_REGION }}
+      - name: Login to Amazon ECR
+        uses: aws-actions/amazon-ecr-login@v1
+      - run: ./build.sh
```

Comment:
Could build.sh be rolled into its own action? E.G. `build-eb-dockerrun`, which could be run after the existing [`build-push-ecr`](https://github.com/mbta/actions/tree/main/build-push-ecr)?

Reply:
It could, but I think each project has their own which works slightly differently at this point?

Reply:
Talked with @ianwestcott on Slack, and I'm going to look into adding an action to generalize this step.